tests/test_app_utils.py

```diff
@@ -24,7 +24,7 @@ def test_init_connection(self):
 
     def test_get_favicon(self):
         """ Tests that given 'mastertest' we get the right url for favicon """
-        expected = self.conn.ff_server + 'static/img/favicon-fs.ico'
+        expected = 'https://data.4dnucleome.org/static/img/favicon-fs.ico'
```

Comment:
Possibly of help to avoid wired strings:
```
>>> from dcicutils.env_utils import public_url_mappings
>>> public_url_mappings('fourfront-cgap')
{'cgap': 'https://cgap.hms.harvard.edu', 'data': 'https://cgap.hms.harvard.edu', 'staging': 'https://staging.cgap.hms.harvard.edu'}
>>> public_url_mappings('fourfront-mastertest')
{'staging': 'http://staging.4dnucleome.org', 'data': 'https://data.4dnucleome.org'}
```